ci/.gitlab-ci.yml

```diff
@@ -19,9 +19,9 @@ stages:
     - trying
   variables:
     GIT_SUBMODULE_STRATEGY: recursive
-    # Custom spack sha used to make an umpire fix available
-    # (https://github.com/msimberg/spack/commit/770d71dff6428876f2cc996f8a0551df36a59e4f)
-    SPACK_SHA: 770d71dff6428876f2cc996f8a0551df36a59e4f
+    # Custom spack sha used to make an umpire fix available + fix the comgr ci pb
+    # (https://github.com/aurianer/spack/commit/f9b6600356a47ab6f4ffd45cef10b3c18e72269c)
+    SPACK_SHA: f9b6600356a47ab6f4ffd45cef10b3c18e72269c
```

Comment:
See https://github.com/eth-cscs/DLA-Future/pull/692#issuecomment-1330867621

Reply:
Just for reference, currently `comgr` is still built.

With the "changes to external" https://github.com/eth-cscs/DLA-Future/pull/692#issuecomment-1330867621 it won't be involved at all (because it's a dep of `rocprim`, which will be provided externally)

In any case, I agree with you that the comment must be removed if it is not relevant anymore (@aurianer knows better the spack branch situation, so I'll let her decide what's correct)